qa/tasks/vstart_runner.py

```diff
@@ -1215,7 +1214,7 @@ def scan_tests(modules):
     max_required_mgr = 0
     require_memstore = False
 
-    for suite, case in enumerate_methods(overall_suite):
+    for suite_, case in enumerate_methods(overall_suite):
```

Comment:
?
```for case in enumerate_methods(overall_suite).values()```

Reply:
suite is already used by `from unittest import suite` so renaming the variable will solve that.

Reply:
maybe better to use unittest.suite in this case?

Reply:
what's wrong with `suite_`?